example/src/main/java/com/midtrans/sample/controller/MobileSdkController.java

```diff
@@ -1,30 +1,84 @@
 package com.midtrans.sample.controller;
 
 import com.midtrans.Config;
-import com.midtrans.ConfigFactory;
 import com.midtrans.httpclient.error.MidtransError;
-import com.midtrans.service.MidtransSnapApi;
+import com.midtrans.httpclient.SnapApi;
+import com.midtrans.httpclient.TransactionApi;
 import org.json.JSONObject;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.Map;
 
+import static com.midtrans.sample.data.Constant.sandboxServerKey;
+
 @RestController
 public class MobileSdkController {
 
```

Comment:
This is for mobile SDK backend (merchant server sample) right? Not for now, but for backlog:
- consider renaming the file/class into more descriptive, maybe:
> MobileSdkBackend

To avoid confusion merchant thinking that this implementation will automatically get them MobileSDK ready. Because they still need to setup Frontend mobile SDK client code on their app.